app/models.py

```diff
@@ -0,0 +1,78 @@
+
+class Base(db.Model):
+    __abstract__ = True  # Indicates that this class should not be created as a table
+    id = db.Column(UUID(as_uuid=True), primary_key=True,
+                   server_default=text("gen_random_uuid()"))
+
+
+class HarvestSource(Base):
+    __tablename__ = 'harvest_source'
+    
+    name = db.Column(db.String, nullable=False)
+    notification_emails = db.Column(ARRAY(db.String))
+    organization_id = db.Column(db.String, nullable=False)
+    frequency = db.Column(db.String, nullable=False)
+    url = db.Column(db.String, nullable=False)
+    schema_type = db.Column(db.String, nullable=False)
+    source_type = db.Column(db.String, nullable=False)
+    harvest_source_id = db.Column(db.String)
+    harvest_source_name = db.Column(db.String)
+    jobs = db.relationship('HarvestJob', backref='source')
+
+class HarvestJob(Base):
+    __tablename__ = 'harvest_job'
+    
+    harvest_source_id = db.Column(UUID(as_uuid=True),
+                                  db.ForeignKey('harvest_source.id'),
+                                  nullable=False)
+    date_created = db.Column(db.DateTime)
+    date_finished = db.Column(db.DateTime)
+    records_added = db.Column(db.Integer)
+    records_updated = db.Column(db.Integer)
+    records_deleted = db.Column(db.Integer)
+    records_errored = db.Column(db.Integer)
+    records_ignored = db.Column(db.Integer)
+    errors = db.relationship('HarvestError', backref='job', lazy=True)
+
+class HarvestError(Base):
+    __tablename__ = 'harvest_error'
+    
+    harvest_job_id = db.Column(UUID(as_uuid=True),
+                               db.ForeignKey('harvest_job.id'),
+                               nullable=False)
+    record_id = db.Column(db.String, nullable=True)
+    record_reported_id = db.Column(db.String)
+    date_created = db.Column(db.DateTime)
+    type = db.Column(db.String)
+    severity = db.Column(db.String)
+    message = db.Column(db.String)
+
+class Organization(Base):
+    __tablename__ = 'organization'
+    
+    name = db.Column(db.String(), nullable=False)
+    logo = db.Column(db.String(), nullable=True)
+
+class HarvestRecord(Base):
+    __tablename__ = 'harvest_record'
+    
+    job_id = db.Column(UUID(as_uuid=True),
+                       db.ForeignKey('harvest_job.id'),
+                       nullable=False) 
+    identifier = db.Column(db.String(), nullable=False)
+    ckan_id = db.Column(db.String(), nullable=False)
+    type = db.Column(db.String(), nullable=False)
+    source_metadata = db.Column(db.String(), nullable=True)
+    name = db.Column(db.String(), nullable=False)
+    owner_org = db.Column(UUID(as_uuid=True),
+                          db.ForeignKey('organization.id'),
+                          nullable=False)
+    author = db.Column(db.String(), nullable=True)
+    author_email = db.Column(db.String(), nullable=True)
+    maintainer = db.Column(db.String(), nullable=True)
+    maintainer_email = db.Column(db.String(), nullable=True)
```

Comment:
i see the point to have ckan_id (indexed) and source_metadata, no point to have other fields.

Reply:
@btylerburton @rshewitt @jbrown-xentity we may need group discussion for the purpose of this table and which fields we need to keep.

Reply:
after discussion, removed extra fields for now.